GSA-TTS/FAC
# TEST SITE Banner

# TEST SITE Banner

Issue: https://github.com/GSA-TTS/FAC/issues/3203

## Changes:
Add a conditional banner to the header that is big, red, and scary, to warn users that they're not in production. The banner displays in all non-production environments, which will include your local. Unless for some reason your `.env` thinks you're in production.

Add `current_environment` to the config context processors, which makes the variable `ENVIRONMENT` available in all HTML templates. 

## How to test:
1. Switch to this branch and run normally. Verify that the banner displays.
2. Swap out your `ENV` environment variable for "PRODUCTION". Re-up docker. Verify that the banner no longer displays. 

## Screenshot:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/76cbcecb-1a74-443c-b092-f3547587f927)

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
